src/read_csv.py

```diff
@@ -3,13 +3,16 @@
 import os
 
 
-def readFile(file):
-    countylist = []
-    newspaperlist = []
+def read_in_counties_newspapers(file):
+    """ A function that takes in a file of counties and the
+    corresponding local newspaper. Then, stores the counties in
+    a list, and the local newspapers in another list and stores
+    the two lists in a tuple."""
+    county_list = []
+    newspaper_list = []
     with open(file, newline='') as csvfile:
         reader = csv.DictReader(csvfile)
         for row in reader:
-            countylist.append(row['counties'])
-            newspaperlist.append(row['localnewspapers'])
-    totallist = (countylist,newspaperlist)
-    return totallist
+            county_list.append(row['counties'])

```

Comment:
Why are you returning a tuple of list? You should return a list of tuples.

Reply:
You still need to fix this. You can do this by saying `newspaper_list.append((row['counties], row['localnewspapers']))` and removing county list. Then you will have a list of `[ (county1, newspaper1), (county2, newspaper2)...]`